bitnami/grafana/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "grafana.fullname" . }}-test-connection"
+  labels:
+    app.kubernetes.io/name: {{ include "grafana.name" . }}
+    helm.sh/chart: {{ include "grafana.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args:  ['{{ include "grafana.fullname" . }}:{{ .Values.service.port }}']
+  restartPolicy: Never
```

Comment:
Why do we need this?

Reply:
We don't need it, It is created by default and I forgot to remove it, sorry.